pytorch_lightning/tuner/batch_size_scaling.py

```diff
@@ -40,6 +40,8 @@ def scale_batch_size(
     batch_arg_name: str = "batch_size",
 ) -> Optional[int]:
     """See :meth:`~pytorch_lightning.tuner.tuning.Tuner.scale_batch_size`"""
+    raise MisconfigurationException("this is gone")
```

Comment:
?

Reply:
just a temporary thing.. will be removing this whole code later.